### PR TITLE
chore: fix podspec

### PR DESCRIPTION
**Pull request by Bang9**
fix source_files path and tag version format
maybe related with #627, but I'm not sure

### CODE REVIEW

File: RNIap.podspec [hunk 1]
```diff
@@ -4,14 +4,13 @@ package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
...
-  s.description  = package['description']
+  s.summary	 = package['description']
   s.homepage     = "https://github.com/dooboolab/react-native-iap"
   s.license      = "MIT"
   s.author       = package['author']
-  s.platform     = :ios, "7.0"
-  s.source       = { :git => "https://github.com/dooboolab/react-native-iap", :tag => "v#{s.version}" }
-  s.source_files  = "ios/**/*.{h,m}"
+  s.platform     = :ios, "9.0"
+  s.source       = { :git => "https://github.com/dooboolab/react-native-iap", :tag => "#{s.version}" }
```
Comment (hyochan) [inline RNIap.podspec]: Could you explain why you removed `v` before `#{s.version}`?

File: RNIap.podspec [hunk 2]
```diff
@@ -4,14 +4,13 @@ package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
...
+  s.source_files  = "ios/*.{h,m}"
```
Comment (hyochan) [inline RNIap.podspec]: Could you also explain how this could fix your problem?

File: RNIap.podspec (hunk 1 above)
Comment (Bang9) [reply RNIap.podspec]: image

This repository using version format as like '3.x.x' not 'v3.x.x'

File: RNIap.podspec (hunk 2 above)
Comment (Bang9) [reply RNIap.podspec]: It is related auto linking
current podspec source_files path is "ios/**/*.{h,m}", but there is no files to link
```
ios/RNIapIos.m
ios/RNIapIos.h
ios/IAPPromotionObserver.h
ios/IAPPromotionObserver.m
= ios/*.{h,m}
```

File: RNIap.podspec (hunk 1 above)
Comment (hyochan) [reply RNIap.podspec]: Thanks for the detail. I wasn't taking care so much on the release tags. :sweat_smile:

File: RNIap.podspec (hunk 2 above)
Comment (hyochan) [reply RNIap.podspec]: Thanks again for this! Looks quite reasonable and I'll merge it for now.

Comment (Bang9) [reply RNIap.podspec]: thanks for confirming!

**Pull request merged**